src/b2b/organizations.rs

```diff
@@ -602,7 +624,7 @@ pub struct SearchRequest {
     /// limit: The number of search results to return per page. The default limit is 100. A maximum of 1000
     /// results can be returned by a single search request. If the total size of your result set is greater than
     /// one page size, you must paginate the response. See the `cursor` field.
-    pub limit: std::option::Option<u32>,
+    pub limit: std::option::Option<i32>,
```

Comment:
Very, *very* sus. This looks wrong.

Reply:
@vincent-stytch I merged the internal codegen change for this, so if you rebase your codegen branch and re-generate it this should go away